packages/compass-components/src/components/workspace-tabs/tab.tsx

```diff
@@ -254,7 +254,7 @@ function Tab({
           tabIndex={isSelected ? 0 : -1}
           aria-controls={tabContentId}
           data-testid="workspace-tab-button"
-          data-connectionName={connectionName}
+          data-connection-name={connectionName}
```

Comment:
small drive-by: we use data-connection-name already in the sidebar, so I changed this to match because it threw me off at first.